Jenkinsfile.deploy

```
@@ -178,10 +228,30 @@ pipeline {
                 }
             }
         }
+        stage('Create PR')
+        {
+            when {
+                expression {
+                    return params.RELEASE
+                }
+            }
+            steps{
+                script{
+                    githubscm.pushObject("origin", NEW_BRANCH , "bsig-gh-bot")
```

`NEW_BRANCH` does not exist in the pipeline. It might be a leftover from earlier.

```suggestion
                    githubscm.pushObject("origin", params.PROJECT_VERSION , "bsig-gh-bot")
```